google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreImpl.java

```diff
@@ -194,28 +194,29 @@ public Iterable<CollectionReference> listCollections() {
   @Override
   public ApiFuture<List<DocumentSnapshot>> getAll(
       @Nonnull DocumentReference... documentReferences) {
-    return this.getAll(documentReferences, null, (ByteString) null);
+    return this.getAll(documentReferences, null, (ByteString) null, null);
```

Comment:
"getAll" functions are called with too many "null"s. Maybe make a overload function call the next overload function that has just one more parameter, and chain them up. e.g, here we call:
`return this.getAll(documentReferences, null);`, 
which should call
`public ApiFuture<List<DocumentSnapshot>> getAll(
      @Nonnull DocumentReference[] documentReferences, @Nullable FieldMask fieldMask) `